http-api/src/main/java/net/runelite/http/api/hiscore/HiscoreEndpoint.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2017, Adam <Adam@sigterm.info>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *  Redistributions of source code must retain the above copyright notice, this
+ *   list of conditions and the following disclaimer.
+ *
+ *  Redistributions in binary form must reproduce the above copyright notice,
+ *   this list of conditions and the following disclaimer in the documentation
+ *   and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package net.runelite.http.api.hiscore;
+
+public enum HiscoreEndpoint
+{
+	NORMAL("normal", "Normal"),
+	IRONMAN("ironman", "Ironman"),
+	HARDCORE_IRONMAN("hardcore_ironman", "Hardcore Ironman"),
+	ULTIMATE_IRONMAN("ultimate_ironman", "Ultimate Ironman"),
+	DEADMAN("deadman", "Deadman"),
+	SEASONAL_DEADMAN("seasonal_deadman", "Seasonal Deadman");
+
+	private final String prettyName;
+
+	HiscoreEndpoint(String name, String prettyName)
+	{
+		this.prettyName = prettyName;
+	}
+
+	public String prettyName() { return prettyName; }
```

Comment:
Make sure to run checkstyle

Reply:
Checkstyle runs but for some reason it isn't catching this...

Reply:
I'm able to get Checkstyle to fail the build if I change [this option](https://github.com/runelite/runelite/blob/master/pom.xml#L43) to false. Is there something configured wrong on my end?